content/techniques/configuration.md

```diff
@@ -310,3 +312,31 @@ export class AppService {
   }
 }
 ```
+
+#### Expandable Variables
+
+The `@nestjs/config` package allows you to take advantage of the **"Expandable Variables"** functionality. You can use your exisiting `.env` variables to declare new variables within `.env` itself. For eg:
+
+```json
+APP_URL=mywebsite.com
+SUPPORT_EMAIL=support@${APP_URL}
+```
+
+> info **Hint** `@nestjs/config` package internally uses [dotenv-expand](https://github.com/motdotla/dotenv-expand).
+
+This can simply be done using the `expandVariables` _(boolean)_ option in the `ConfigModule` like this:
+
+```typescript
+@@filename(app.module)
+@Module({
+  imports: [
+    ConfigModule.forRoot({
+      // ...
+      expandVariables: true,
+    }),
+  ],
+})
+export class AppModule {}
+```
+
+This will convert the value of the variable `SUPPORT_EMAIL` to `support@mywebsite.com` and make it available in `process.env` as `process.env.SUPPORT_EMAIL`.
```

Comment:
Suggestion: Replace 
"This will convert the value of the variable `SUPPORT_EMAIL` to `support@mywebsite.com` and make it available in `process.env` as `process.env.SUPPORT_EMAIL`."

with

"With this construction, the variable `SUPPORT_EMAIL` resolves to `'support@mywebsite.com'`.  Note the use of the `${...}` syntax to trigger resolving the value of the variable `APP_URL` inside the definition of `SUPPORT_EMAIL`.

Also, I suggest moving this immediately below the example itself.

Reply:
@johnbiundo Done 👍

Reply:
Ahh, that's a problem with the djeni compiler.  You need to escape the { and }.  Can you try the following please:

${{ '{' }}...{{ '}' }}

I **think** that's the way you do it, but if not, I guess we could go with something like:

"With this construction, the variable SUPPORT_EMAIL resolves to 'support@mywebsite.com'. Note the use of the following syntax to trigger resolving the value of the variable APP_URL inside the definition of SUPPORT_EMAIL:
```typescript
${...}
```

Reply:
@johnbiundo Awesome! That fixed it. Thanks! We're good to go 👍